Provide draco-7_8_0.

KineticTheory opened the pull request:
+ Also provide a patchfile for draco-7_6_0 to support CrayPE builds.
+ Version 7.8.0 has a new variant `+caliper`.
+ Sort dependencies alphabetically after grouping by required and optional.

KineticTheory committed:
Provide draco-7_8_0.

+ Also provide a patchfile for draco-7_6_0 to support CrayPE builds.
+ Version 7.8.0 has a new variant `+caliper`.
+ Sort dependencies alphabetically after grouping by required and optional.

alalazo commented on var/spack/repos/builtin/packages/draco/package.py:
```diff
@@ -64,13 +68,14 @@ class Draco(CMakePackage):
     patch('d730.patch', when='@7.3.0:7.3.99')
     patch('d740.patch', when='@7.4.0:7.4.99')
-    patch('smpi.patch', when='@:7.6.99')
+    patch('d760-cray.patch', when='@7.6.0')
```
Is `smpi.patch` used at all now? should we delete it from the repository?

KineticTheory replied:
> Is `smpi.patch` used at all now? should we delete it from the repository?

@alalazo That looks like a bug on my part.  Let me recheck the whole file to make sure I didn't mess something else up.  Thanks for noticing!

KineticTheory replied:
@alalzo Indeed, older versions of draco are no longer supported for `^spectrum-mpi` and newer versions work without the patch.  I've deleted the excess file in my updated PR.

alalazo reviewed (commented):
Just a question on the new patch, otherwise LGTM.

KineticTheory committed:
Remove patchfile that is no longer needed.

+ Newer versions of draco do not require this patch.
+ Older versions of draco are not supported for spectrum-mpi.

KineticTheory committed:
Change new variant +caliper to default to False.
